pom.xml

```diff
@@ -157,6 +148,10 @@ since this is impossible for AWS S3 dependencies without Maven.
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-war-plugin</artifactId>
                 <version>3.3.2</version>
+                <configuration>
+                    <warSourceDirectory>${project.basedir}</warSourceDirectory>
+                    <warSourceExcludes>WEB-INF/lib/**, target/**, data/**, content/**, .settings/**, *.*</warSourceExcludes>
```

Comment:
If I'm reading the makeErddapWar.bat properly, it includes WEB-INF/lib/ in the inputs. Is excluding this an intentional change?

Reply:
Yes, intentional, but it could be left in or taken out, it won't matter in the resulting WAR file generated.  Maven copies the same libraries to `target/${project.build.finalName}/WEB-INF/lib` based on the dependency configuration in pom.xml if `WEB-INF/lib` is excluded here.  I've tested several times to confirm.  

Actually, the `WEB-INF/lib` directory in the ERDDAP repo could be removed of all JAR files, except `netcdfAll-5.5.3.jar` (since it's not available elsewhere any longer) if desired.  Then pom.xml would be entirely responsible for determining dependencies.  I think this is better myself, but it's not essential.